cmd/scip/tests/reprolang/package.json

```diff
@@ -12,6 +12,6 @@
     "nan": "^2.15.0"
   },
   "devDependencies": {
-    "tree-sitter-cli": "^0.20.4"
+    "tree-sitter-cli": "0.21.0"
```

Comment:
Minimal version that is both deterministic (https://github.com/tree-sitter/tree-sitter/issues/2755), available from arm64 (for local work), **and** available on NPM: https://www.npmjs.com/package/tree-sitter-cli?activeTab=versions (0.20.9 is missing)

Reply:
It's for tests only, so doesn't matter